api_versions.yml

```diff
@@ -95,7 +95,7 @@ admin:
     conversion_dispositions: 2
     decrypt_affiliate_link: 1
     mass_conversion_adjustment: 2
-    mass_conversion_insert: 2
+    mass_conversion_insert: 1
```

Comment:
Version 2 returns a "Unable to handle request without a valid action parameter. Please supply a valid soap action." error when using it's WSDL... Version 1 works fine for the same request.